helix-core/src/test/java/org/apache/helix/integration/TestErrorReplicaPersist.java

```diff
@@ -103,10 +103,9 @@ public void testErrorReplicaPersist() throws InterruptedException {
     ClusterConfig clusterConfig = configAccessor.getClusterConfig(CLUSTER_NAME);
     clusterConfig.setErrorPartitionThresholdForLoadBalance(100000);
     configAccessor.setClusterConfig(CLUSTER_NAME, clusterConfig);
-    
+
     for (int i = 0; i < (NODE_NR + 1) / 2; i++) {
       _participants[i].syncStop();
-      Thread.sleep(2000);
```

Comment:
Better not remove this, this wait time is for cluster converge. If you need to fix this, would be good to put a verifier here.